HackerRank Problem/Python/GradingStudents.py

```diff
@@ -0,0 +1,44 @@
+import math
+import os
+import random
+import re
+import sys
+
+#
+# Complete the 'gradingStudents' function below.
+#
+# The function is expected to return an INTEGER_ARRAY.
+# The function accepts INTEGER_ARRAY grades as parameter.
+#
+
+
+def gradingStudents(grades):
+    op_grades = []
+    for grade in grades:
+        ng = grade
+        if grade >= 38 and (grade % 5 != 0):
+            for num in (1,2):
+                new_grade = grade + num
+                if new_grade % 5 == 0:
+                    ng = new_grade
+        op_grades.append(ng)
+    return op_grades
+
+
+if __name__ == '__main__':
+    fptr = open(os.environ['OUTPUT_PATH'], 'w')
+
+    grades_count = int(input().strip())
+
+    grades = []
+
+    for _ in range(grades_count):
+        grades_item = int(input().strip())
+        grades.append(grades_item)
+
+    result = gradingStudents(grades)
+
+    fptr.write('\n'.join(map(str, result)))
+    fptr.write('\n')
+
+    fptr.close()
```

Comment:
Remove extra lines after each statements

Reply:
I've made the changes @aneels3